dwitter/templates/dweet/dweet.html

```diff
@@ -277,31 +278,10 @@
     }
     
     function setupPolyfills(x) {
-      c._width = c.width;
-      Object.defineProperty(c, "width", {
-        set: function(value) {
-          x.beginPath();
-          x.resetTransform();
-          x.clearRect(0, 0, c.width, c.height);
-          c._width = value;
-        },
-        get: function() {
-          return c._width;
-        },
-        configurable: true
-      });
-      c._height = c.height;
-      Object.defineProperty(c, "height", {
-        set: function(value) {
-          x.beginPath();
-          x.resetTransform();
-          x.clearRect(0, 0, c.width, c.height);
-          c._height = value;
-        },
-        get: function() {
-          return c._height;
-        },
-        configurable: true
+      $(c).resize(function() {
+        x.beginPath();
+        x.resetTransform();
+        x.clearRect(0, 0, obj.width, obj.height);
```

Comment:
So `'resize'` gets triggered even if the canvas didn't change size? wtfdom!

Reply:
Mmm yeah, regarding @lionleaf's performance question, will this `clearRect` run on all platforms, whenever `c.width` is assigned?

Won't that impact performance on platforms where it's not needed?